### DISCUSSION TITLE
Importing Beatmaps to osu!lazer for Android

### DISCUSSION THREAD

**Discussion by EPICNESS2008** [ppy/osu#29345] [Q&A]
I just recently got osu!lazer for my phone and I don't know how to import beatmaps to my phone. I tried going through my phones files when I came across the files page, I didn't touch the files, as what the README said. How do I do this?

**Comment by LumpBloom7** [comment]
you can use the share button in your file manager to share the files to osu, which will import the beatmap / skin
